PizzaBuilder/Program.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+namespace Builder
+{
+    public interface IPizzaBuilder
+    {
+        void BuildDough();
+        void BuildSauce();
+        void BuildTopping();
+    }
+    public class WhitePizzaBuilder : IPizzaBuilder
+    {
+        private Pizza _pizza = new Pizza();
+        public WhitePizzaBuilder()
+        {
+            this.Reset();
+        }
+        public void Reset()
+        {
+            this._pizza = new Pizza();
+        }
+        public void BuildDough()
+        {
+            this._pizza.Add("Dough");
+        }
+        public void BuildSauce()
```

Comment:
It would be nice if you specified some sauces like (White sauce, red sauce) or what you want rather than just writing the word Sausce.